docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,665 @@
+        image_features=False,
+        meta_features="infer",
+        method="standardize",
+        samples="Metadata_control_type == 'negcon'",
+        output_file=parquet_pycytominer_normalized_wo_outliers,
+        output_type="parquet",
+    )
+
+# +
+parquet_pycytominer_feature_selected_wo_outliers = (
+    f"./{example_plate}_feature_select_wo_outliers.parquet"
+)
+
+# check if we already have feature selected data
+if not pathlib.Path(parquet_pycytominer_feature_selected_wo_outliers).is_file():
+    # feature select normalized data using pycytominer
+    df_pycytominer_feature_selected = pycytominer.feature_select(
+        profiles=parquet_pycytominer_normalized_wo_outliers,
+        operation=[
+            "variance_threshold",
+            "correlation_threshold",
+            "blocklist",
+            "drop_na_columns",
+        ],
+        na_cutoff=0,
+        output_file=parquet_pycytominer_feature_selected_wo_outliers,
+        output_type="parquet",
+    )
+# -
+
+# calculate UMAP embeddings from data without coSMicQC-detected outliers
+embeddings_without_outliers = generate_umap_embeddings(
+    df_input=pd.read_parquet(parquet_pycytominer_feature_selected_wo_outliers),
+    cols_metadata_to_exclude=all_metadata_cols,
+)
+# show the shape and top values from the embeddings array
+print(embeddings_without_outliers.shape)
+embeddings_without_outliers[:3]
+
+# plot UMAP for embeddings without outliers
+plot_hvplot_scatter(
+    embeddings=embeddings_without_outliers,
+    title=f"UMAP of JUMP embeddings from {example_plate} (without erroneous outliers)",
+    filename=(
+        image_without_all_outliers := f"./images/umap_without_outliers_{example_plate}.png"
+    ),
+    bgcolor="white",
+    cmap="coolwarm",
+)
+
+# compare the UMAP images with and without outliers side by side
+HTML(
+    f"""
+    <div style="display: flex;">
+      <img src="{image_with_all_outliers}" alt="UMAP which includes erroneous outliers" style="width: 50%;"/>
+      <img src="{image_without_all_outliers}" alt="UMAP which includes no erroneous outliers" style="width: 50%;"/>
+    </div>
+    """
+)
+
+# +
+# concatenate embeddings together
+combined_embeddings = np.vstack((embeddings_with_outliers, embeddings_without_outliers))
+
+# Step 2: Create the labels array
+combined_labels = np.concatenate(
+    [np.zeros(len(embeddings_with_outliers)), np.ones(len(embeddings_without_outliers))]
+)
+
+# visualize UMAP embeddings both with and without outliers together for comparison
+plot_hvplot_scatter(
+    embeddings=combined_embeddings,
+    title=f"UMAP comparing JUMP embeddings from {example_plate} with and without erroneous outliers",
+    filename=f"./images/umap_comparison_with_and_without_erroneous_outliers_{example_plate}.png",
+    color_dataframe=pd.DataFrame(
+        combined_labels, columns=["combined_embedding_color_label"]
+    ),
+    color_column="combined_embedding_color_label",
+    bgcolor="white",
+    cmap="RdYlGn",
```

Comment:
a couple comments here, that need not delay merging (you probably don't need to address either, since my understanding is the UMAP you will show in the poster is the `coolwarm` palette. 

- We should avoid Red/Green color palette to accomodate folks with this colorblindness
- There are only three categories correct? Then shouldn't we use a different legend type?
- In general, all the legends should be labelled.

Reply:
Thanks! I'll correct the color usage with vision accessibility in mind.

> There are only three categories correct? Then shouldn't we use a different legend type?

Because of how these plots display points it's important to provide gradients to show "less" or "more" based on how much information a pixel can contain. That said, I think you're correct, that these should only display two colors with their matching gradients.

> In general, all the legends should be labelled.

Thanks for the added attention to this, I had trouble finding information on this (as per #90 ) but found that this is called something different in Bokeh plots today (colomap labels). I've added labels to the function and every scatter plot in a later commit which will be added.